### DISCUSSION TITLE
Integration with systemd

### DISCUSSION THREAD

**Discussion by mauvray** [nwg-piotr/nwg-shell#303] [Ideas]
Hello. Firsrt thing first, nwg-shell is nice. I am still begining with it but I like it.
What do you think about systemd integration when available instead of the autostart ? I think there is some documentation in the sway wiki (can't check immediately)

**Comment by nwg-piotr** [comment]
Hi,

Honestly, I don't think it's a good idea. With what we already have, we make life difficult enough for packagers on systemd-less distros.

**Reply by mauvray** [reply]
I can understand. In that case, is it possible to add a toggle to not use the autostart file if I want to setup systemd integration on my computer ?

**Reply by nwg-piotr** [reply]
You are on sway, right? If so, just delete `include autostart` from the main config file.
